pkgs/applications/audio/dragonfly-reverb/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchgit, libjack2, fftwFloat, libGL, lv2, pkgconfig, xorg }:
+
+stdenv.mkDerivation rec {
+  name = "dragonfly-reverb-${src.rev}";
+
+  src = fetchgit {
+    url = "https://github.com/michaelwillis/dragonfly-reverb";
+    rev = "0.9.0";
+    deepClone = true;
```

Comment:
This option is evil and will break sooner or later hashes:

```
Initialized empty Git repository in /nix/store/jawvawgdp3v30l3i640z1lnbiqp47421-dragonfly-reverb/dpf/.git/
remote: Counting objects: 7135, done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 7135 (delta 4), reused 1 (delta 0), pack-reused 7123
Receiving objects: 100% (7135/7135), 12.13 MiB | 1.50 MiB/s, done.
Resolving deltas: 100% (6046/6046), done.
From https://github.com/DISTRHO/DPF
 * [new branch]      gh-pages       -> origin/gh-pages
 * [new branch]      master         -> origin/master
 * [new branch]      vst-key-events -> origin/vst-key-events
Switched to a new branch 'fetchgit'
Deleted remote-tracking branch origin/master (was bfd41a9).
Deleted remote-tracking branch origin/simplify-parameters (was 55a51f1).
Deleted remote-tracking branch origin/spectrogram (was d021e57).
Deleted remote-tracking branch origin/gh-pages (was c159351).
Deleted remote-tracking branch origin/master (was a50ea21).
Deleted remote-tracking branch origin/vst-key-events (was 0393d1e).
fixed-output derivation produced path '/nix/store/8zcmnqaf86z4rhggwf86hnkqyakjxagm-dragonfly-reverb' with sha256 hash '1dn54xs2zllrpfjd2870g8z8pzyhn9w7rmqvrr50zy9zqzlirvv1' instead of the expected hash '1hji62sdf87sz38qgk7m6r42ak2a6s59q387qiakhnnykh3bp7vg'
cannot build derivation '/nix/store/wjygxhias169ghj4gjdn56fnbczx7f3y-dragonfly-reverb-0.9.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/wjygxhias169ghj4gjdn56fnbczx7f3y-dragonfly-reverb-0.9.0.drv' failed
The invocation of 'nix-shell --no-out-link --keep-going --max-jobs 4 --option build-use-sandbox true --run true -p dragonfly-reverb' failed
```

Is it possible to workaround the dependency on .git?

Reply:
I was able to build the packages without this option.